### PR TITLE
[Documentation] More explicit platform.sh guide

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/cookbook/platform-sh.rst [hunk 1]
```diff
@@ -201,8 +201,14 @@ Remember to have it imported in the config:
 
     # app/config/config.yml
     imports:
+        # - { resource: parameters.yml } <- Has to be place before our new file
         - { resource: parameters_platform.php }
 
+.. warning::
+
+    It is important to place newly created file after importing regular parameters.yml file. Otherwise your database connection will not work.
+Also this will be the file where you should set required parameters for your. Theirs value will be fetched from environmental variables.
```
Comment (CoderMaggie) [inline docs/cookbook/platform-sh.rst]: 1. Indent is missing :)
2. `... set required parameters for your.` for your what? 😄 
3. `Theirs value...` => `Its value...`

File: docs/cookbook/platform-sh.rst [hunk 2]
```diff
@@ -201,8 +201,14 @@ Remember to have it imported in the config:
 
     # app/config/config.yml
     imports:
+        # - { resource: parameters.yml } <- Has to be place before our new file
```
Comment (CoderMaggie) [inline docs/cookbook/platform-sh.rst]: why do you have this line commented?
`Has to be place...` => `Has to be placed...`

File: docs/cookbook/platform-sh.rst [hunk 3]
```diff
@@ -242,7 +248,13 @@ When you get connected please run:
 
 .. code-block:: bash
 
-    $ php bin/console sylius:install
+    $ php bin/console sylius:install --env prod
+
+.. warning::
+
+    By default platform.sh creates only one instance of database with the `main` name. Installing Sylius on dev environment would required
+`main_dev` which of course will end up with an exception. Deploying development environment is possible but will require a little bit more
```
Comment (CoderMaggie) [inline docs/cookbook/platform-sh.rst]: also here indent is missing. All text that you want to be in the warning, has to be indented.

Comment (stefandoorn) [inline docs/cookbook/platform-sh.rst]: I think it's good to note that Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch. It's rather hard to activate per branch prod/dev settings in my experience until now using Platform.sh.

Comment (CoderMaggie) [inline docs/cookbook/platform-sh.rst]: @lchrusciel would you be so kind to describe it? :)

Comment (lchrusciel) [inline docs/cookbook/platform-sh.rst]:
```
 By default platform.sh creates only one instance of a database with the `main` name. Platform.sh works with the concept of an environment per branch if activated. The idea is to mimic production settings per each branch.
```
would be fine?

Comment (stefandoorn) [inline docs/cookbook/platform-sh.rst]: Sure. I'm planning to add some more instructions to this document later based on my findings using Platform in production, but haven't had time to do it yet..

**Comment by CoderMaggie** [comment]
Thank you Łukasz! 🥇

**Pull request merged**